__docs__/wonder-blocks-icon/accessibility.stories.mdx

```diff
@@ -0,0 +1,99 @@
+import {Meta, Story, Canvas} from "@storybook/addon-docs";
+
+import Color from "@khanacademy/wonder-blocks-color";
+import Icon, {icons} from "@khanacademy/wonder-blocks-icon";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
+import {LabelMedium} from "@khanacademy/wonder-blocks-typography";
+
+<Meta
+    title="Icon / Accessibility"
+    component={Icon}
+    parameters={{
+        previewTabs: {
+            canvas: {hidden: true},
+        },
+        viewMode: "docs",
+        chromatic: {
+            // Disables chromatic testing for these stories.
+            disableSnapshot: true,
+        },
+    }}
+/>
+
+## Icon Accessibility
+
+### Rules of Thumb
+
+-   Use familiar or commonly used icons.
+-   Each icon should convey a single meaning.
+-   Icons should be easy to see.
+-   The color contrast should pass WCAG.
+    -   At least 4.5:1 contrast ratio for icons.
+-   Ideally, an icon should be accompanied by text. However, if an icon
+    is not accompanied by text, provide alt text.
+    -   Do not provide alt text for decorative icons. Set the alt text
+        to an empty string (`""`).
+    -   The `title` attribute is often used to display a description on mouse
+        hover, but please note that use of the `title` attribute is discouraged
+        as titles are not available to keyboard-only or touch-only users.
```

Comment:
**thought:** What do you think about mentioning that there's more info in the `References` section?

Reply:
Did you mean this specifically for this bullet point? Or for the entire list?

Reply:
yep, specifically to this bullet point.